docs/5.x/overview.md

```diff
@@ -40,6 +40,33 @@ their dependencies, a private Docker registry for autonomous operation, a
 monitoring system and an SSH bastion for remotely managing the cluster either
 via simple SSH or via Kubernetes API.
 
+
+## Components
+
+Gravity consists of these major components:
+
+
+* [gravity](./cluster/#gravity-tool): the CLI tool used to manage the cluster. Gravity is only available inside the cluster so you have to `tsh ssh` into the cluster to execute `gravity` commands.
+* **`tele`**: the CLI tool which is used for packaging and publishing applications.
+* **`tsh`**: the SSH client for establishing secure SSH connections between the
+  Ops Center and the application instances running behind firewalls on private clouds.
+  `tsh` is a part of [Gravitational Teleport](http://gravitational.com/teleport/),
+  a free open source SSH server developed, maintained and supported by Gravitational.
+  It can be used independently from Gravity.
+* [Ops Center](./opscenter): the Web UI for managing published applications and remotely
+  accessing private cloud deployments.
+
+## Workflow
+
+The typical Gravity workflow is as follows:
+
+* Start by building and publishing your Application Bundle with `tele` command.  Once
+  a Gravity Cluster is deployed, `tele` command will let you list the active
```

Comment:
I would not emphasize (i.e. capitalize) `Cluster` - `Gravity cluster is deployed.` Ditto for other places.